components/mintmaker/development/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base
+  - https://github.com/konflux-ci/mintmaker/config/default?ref=9e3dd3bfec43fbbae2d2dfb931f3f3cb082ae9af
```

Comment:
The manifest brought by this revision do have kube linter issues, this needs to be fixed as we have a check for that, which is failing. Pasting the issues here so you do not have to drill down in the check logs:
```
<standard input>: (object: mintmaker/mintmaker-controller-manager apps/v1, Kind=Deployment) container "kube-rbac-proxy" does not have a read-only root file system (check: no-read-only-root-fs, remediation: Set readOnlyRootFilesystem to true in the container securityContext.)

<standard input>: (object: mintmaker/mintmaker-controller-manager apps/v1, Kind=Deployment) container "manager" does not have a read-only root file system (check: no-read-only-root-fs, remediation: Set readOnlyRootFilesystem to true in the container securityContext.)
```